ranger-plugin/admin-plugin/src/main/java/org/apache/hawq/ranger/service/RangerServiceHawq.java

```diff
@@ -112,7 +119,7 @@ private void decryptPassword(Map<String, String> configs) throws Exception {
                 // when decrypt failed do nothing
                 LOG.warn("decrypt_password failed: " + e);
             }
-            configs.put("password", normal_password);
+            configs.put("password_jdbc", normal_password);
```

Comment:
put the decrypted password into a new field, so it doesn't influence the BaseClient decrypt it again.